packages/MSBot/src/msbot-clone.ts

```diff
@@ -87,22 +99,38 @@ async function processConfiguration(): Promise<void> {
         }
 
         logCommand(args, `Fetching subscription account`, command);
-        p = await exec(command);
-        let azAccount = JSON.parse(p.stdout);
+        output = await exec(command);
+        const azAccount: {id: string; tenantId: string; name: ''} = JSON.parse(output.stdout);
         args.subscriptionId = azAccount.id;
         args.tenantId = azAccount.tenantId;
         if (!args.quiet) {
             console.log(`Creating resources in subscription: ${azAccount.name} ${azAccount.id}`);
         }
 
         // create group
-        let azGroup: any;
+        const azBase: ICloneArgs = {
+            name: '',
+            folder: '',
+            location: '',
+            subscriptionId: '',
+            tenantId: '',
+            groupName: '',
+            secret: '',
+            quiet: false,
+            verbose: false,
+            luisAuthoringKey: '',
+            luisSubscriptionKey: '',
+            qnaSubscriptionKey: '',
+            luisRegion: '',
+            args: []
+        };
```

Comment:
you can't do this.  This entire function is written on the assumptiotn that there ARE NO values.  You can't introduce values without breaking all of the logic.